packages/@dcl/sdk-commands/src/logic/bundle.ts

```diff
@@ -170,3 +206,51 @@ function runTypeChecker(components: BundleComponents, options: CompileOptions) {
 
   return typeCheckerFuture
 }
+
+function compositeLoader(composites: Record<string, Uint8Array>): esbuild.Plugin {
+  const compositeLines: string[] = []
+
+  for (const compositeName in composites) {
+    const bin = composites[compositeName]
+    if (compositeName.endsWith('.bin')) {
+      compositeLines.push(`'${compositeName}':new Uint8Array(${JSON.stringify(Array.from(bin))})`)
+    } else {
+      const textDecoder = new TextDecoder()
+      const json = JSON.stringify(JSON.parse(textDecoder.decode(bin)))
+      compositeLines.push(`'${compositeName}':'${json}'`)
+    }
+  }
+
+  const contents = `export const compositeFromLoader = {${compositeLines.join(',')}}`
```

Comment:
(cont) pseudo code from my previous suggestion


```suggestion
  // this will contain the parsed composite objects
  const compositeObjects: Record<string, any> = {}

  for (const compositeName in composites) {
    const bin = composites[compositeName]
    const textDecoder = new TextDecoder()
    compositeObjects[compositeName] = JSON.parse(textDecoder.decode(bin))
  }

  const contents = `export const compositeFromLoader: Record<string, CompositeType> = ${JSON.stringify(compositeObjects)}`
```